applications/volumemanager/server/Dockerfile

```diff
@@ -1,21 +1,20 @@
 ARG REGISTRY
 ARG TAG=latest
-FROM ${REGISTRY}cloudharness-base:${TAG}
+ARG CLOUDHARNESS_FLASK=${REGISTRY}cloudharness-flask:${TAG}
 
-RUN apk update
-RUN apk upgrade
+FROM $CLOUDHARNESS_FLASK as frontend
 
-RUN mkdir -p /usr/src/app
-WORKDIR /usr/src/app
+ENV MODULE_NAME=volumemanager
+
+ENV WORKERS=2
+ENV PORT=8080
 
 COPY requirements.txt /usr/src/app/
 
 RUN pip3 install --no-cache-dir -r requirements.txt
 
 COPY . /usr/src/app
 
-EXPOSE 8080
-
-ENTRYPOINT ["python3"]
+RUN pip3 install -e .
```

Comment:
why install the working folder in edit mode?

Reply:
It's simpler to track the debugger files mapping if we don't move the files